static/app/views/settings/organizationTeams/teamMembers.tsx

```diff
@@ -263,9 +264,15 @@ class TeamMembers extends AsyncView<Props, State> {
         onChange={this.handleMemberFilterChange}
         busy={this.state.dropdownBusy}
         onClose={() => this.debouncedFetchMembersRequest('')}
+        disabled={team?.flags['idp:provisioned']}
```

Comment:
Nit: I don't think you need the `?` because `team` is guaranteed to be there.

Reply:
teamMembers.spec.jsx fails if the ? isn't there

Reply:
i think the `team` loads slower than the component loads, so for a little bit, team is `undefined` and the test fails if there's no `?`